shell/common/animator.cc

```diff
@@ -107,6 +120,46 @@ void Animator::BeginFrame(fml::TimePoint frame_start_time,
     TRACE_FLOW_END("flutter", "PointerEvent", trace_flow_id);
     trace_flow_ids_.pop_front();
   }
+  while (!pointer_data_.empty()) {
+    PointerData pointer_data = pointer_data_.front();
```

Comment:
No need to copy this, right?  We can probably just make this a "const PointerData&"?

Reply:
Yep, done.